src/common.h

```diff
@@ -385,6 +442,7 @@ static LibraryStatistics computeLibraryStats(string bamFileName, uint64_t genome
 	library.C_W = C_W = wronglyOrientedReadsLength/(float)genomeLength;
 	library.C_S = C_S = singletonReadsLength/(float)genomeLength;
 	library.C_D = C_D = matedDifferentContigLength/(float)genomeLength;
+	library.readLength= mappedReadsLength/float(mappedReads);
```

Comment:
`library.readLength` is an int, so it is always better to cast the result of a division with a float to avoid surprises. 
Given how you calculate this number I suggest to rename the variable in `avarageReadLength`